### ISSUE TITLE
Check full installation

### ISSUE THREAD

**Issue by A888R** [Patrowl/PatrowlManager#5]
Please, check new full installation!
After run scan over nmap engines - task not executed.
Test in localhost, API URL: http://localhost:5001/engines/nmap/ state:Ready
RabbitMQ and Celery (flower) see task, but, to http://localhost:5001/engines/nmap/ does not request!

**Comment by MaKyOtOx** [comment]
The [installation guide](https://github.com/Patrowl/PatrowlDocs) has been updated. We've tested it on an Ubuntu 18.04.
I have few questions:
- Did you disable your local firewall ? or are you confident with the rules ?
- Have you activated the engine (see http://patrowlmanager:8000/engines/list). As you probably noticed, there are 2 states: "functional state" that you decide to enable/disable and "operational state" that display the actual availability of the engine. Are the values correct ?
- Is the nmap engine correctly configured in the PatrowlManager ? See a correct configuration: 
<img width="679" alt="capture d ecran 2018-07-10 a 22 14 41" src="https://user-images.githubusercontent.com/1262702/42535113-dc468ce4-848e-11e8-907f-d8eff51c72fd.png">
- Have you correctly configured the nmap scan policy? See a correct configuration:
<img width="656" alt="capture d ecran 2018-07-10 a 22 18 09" src="https://user-images.githubusercontent.com/1262702/42535273-3abf646c-848f-11e8-99cd-260f6b808c71.png">
- Have you correctly configured the Nmap engine ? Review `nmap.json` file and focus on the path where nmap is installed. Nmap is not installed with the Engine (excepted in the the Docker image)

Please share your reviews :)

**Comment by A888R** [comment]
Hi.
1) Manager and Engine in one host (localhost) and firewall not working;
2) ![image](https://user-images.githubusercontent.com/8194134/42540913-ff3ed0ca-84a8-11e8-8874-cc68e725666c.png)
3) yep;
4) I did not touch poliсy;
5) Yep, nmap installed (whereis nmap: /usr/bin/nmap)
I think this is a problem on my part (maybe version in requests.txt?). I create a new virtual machine and test on the next weekend. Thx

**Comment by MaKyOtOx** [comment]
If the Operational state is "Ready", it means PatrowlManager was able to use the full pipe Celery + RabbitMQ (which is a good news).
- Is the nmap engine receiving the "startscan" API call (http://localhost:5001/engines/nmap/startscan) ? See in the stdout in the terminal used to start the engine
- Ensure you use the python module celery==4.0.2 (very important for compatibility)
- Could be send a screenshot of the scan definition. I already now, the edit form is ugly ;)

**Comment by A888R** [comment]
1) Open over browser:
![image](https://user-images.githubusercontent.com/8194134/42597890-1256e5ac-8563-11e8-876a-c4b9431b2cb0.png)
2) No. Celery 4.2.0;
3) ![image](https://user-images.githubusercontent.com/8194134/42598014-7cb4167c-8563-11e8-9327-f5f766d15faa.png)

**Comment by MaKyOtOx** [comment]
Thx ! 
We have an issue with the request. The Manager seems to send a GET resquest which is not allowed (code 405). It should be a POST request :/
Did you change something in engines/tasks.py ?
I’ll be investigating other ideas. 

@fix in the meantime: set the status « ERROR » to the scan in that use case.

**Comment by A888R** [comment]
No! Code 405 - it's correct. I open link over my **browser** (I create this GET-request). This request not from PatrowlManager. Problem: celery task (nmap) not create POST-request to the API nmap engine. Sorry for my bad English.

**Comment by MaKyOtOx** [comment]
No problem with your English :)
Could you please install and use the celery module in version **4.0.2** ? The version 4.2.0 seems to be buggy. Don't forget to restart the supervisord (or restart all celery workers).
Another stupid question: are you sure to use Python2.7 for the Manager and the Engines ? Because it really does not work with Python3.

**Comment by A888R** [comment]
I'm created new VM, installed all packages from _requirements.txt_ (pip install -r requirements.txt) in virtual env. This did not solve the problem (celery **4.0.2**).
Yep, I use in this VM only python2.7.

**Comment by MaKyOtOx** [comment]
Hi, are you able to reproduce the issue with the new version ?
Thanks a lot!
